tools/log-collector/log_collector.go

```diff
@@ -361,7 +362,14 @@ func (l *LogCollector) filterResourceObjects(resourcePath string,
 		}
 	}
 
-	objects = filterTvkResourcesByLabel(allObjects)
+	if nonLabeledResources.Has(resource.Kind) {
+		objects.Items = append(objects.Items, allObjects.Items...)
```

Comment:
no need of append operation and  `objects` var.. just return `allObjects` var directly.

Reply:
resolved